pixart.js

```diff
@@ -1 +1,30 @@
 
+$(document).ready(function canvas() {
+    for (i = 0; i<= 5000; i++)
+{
+
+  $( "body" ).append($("<div class='square'></div>"))
+}
+      this.squares = $(".square")
+});
+
+var inputColor = {
+    this.color = this.input.val()
```

Comment:
What does `this` equal?

Also, if you are inside of an object, you need to make sure to use proper key-value notation.